ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/commands/CommandClient.scala

```diff
@@ -138,10 +138,13 @@ final class CommandClient(
         .via(commandUpdaterFlow[Context])
         .viaMat(
           CommandTrackerFlow[Context, NotUsed](
-            CommandSubmissionFlow[(Context, String)](submit(token), config.maxParallelSubmissions),
-            offset => completionSource(parties, offset, token),
-            ledgerEnd.getOffset,
-            () => Some(config.defaultDeduplicationTime),
+            commandSubmissionFlow = CommandSubmissionFlow[(Context, String)](
+              submit(token),
+              config.maxParallelSubmissions,
+            ),
+            createCommandCompletionSource = offset => completionSource(parties, offset, token),
+            startingOffset = ledgerEnd.getOffset,
+            maximumExpiryTime = config.defaultDeduplicationTime,
```

Comment:
Is there / will there be a way for the client to use a different value than the default deduplication time and why?

Reply:
Yeah, I was wondering this too. I'm not really sure how this is used. It looks to me as if this is the intended behavior based on the Scaladoc in `CommandClientConfiguration`.

We don't reuse trackers in the client (we do on the server), so in this case, the behavior should not have changed at all.

The relevant Scaladoc is as follows:

```scala
/** ...
  * @param defaultDeduplicationTime The deduplication time to use for commands that do not have
  *                                 a deduplication time set. The deduplication time is also used
  *                                 as the time after which commands time out in the command client.
  */
```

Reply:
I wonder if this should be notified and this overlap reconsidered. In this PR, I am fine with leaving things as they are though.